server/routes.js

```diff
@@ -54,7 +54,7 @@ module.exports = (app) => {
   app.get('/:slug/:tier/banner.md', mw.cache(300), mw.fetchGroupBySlug, mw.fetchUsers(), banner.markdown);
   app.get('/:slug/:tier/badge.svg', mw.cache(300), mw.fetchUsers(), banner.badge);
   app.get('/:slug/badge/:tier.svg', mw.cache(300), mw.fetchUsers(), banner.badge);
-  app.get('/:slug/:tier/:position/avatar(.png)?(.jpg)?', mw.cache(300), mw.ga, mw.fetchUsers({cache: 300}), banner.avatar);
+  app.get('/:slug/:tier/:position/avatar(.:format)?', mw.cache(300), mw.ga, mw.fetchUsers({cache: 300}), banner.avatar);
```

Comment:
You are right. But `req.params[0]` is not great to read in the code. Here is a better way to do it:

    app.get('/:slug/:tier/:position/avatar(.:format(png|jpg|svg))?', (req, res) => {
      console.log(req.params.format);
    })